k8s/lib/scripts/configure_k8s_weave.sh

```diff
@@ -8,7 +8,8 @@ function patch_kube_proxy(){
 }
 
 function setup_k8s_weave() {
-    kubectl apply -f $HOME/setup/weave/weave-daemonset-k8s-1.6.yaml
+    export kubever=$(kubectl version | base64 | tr -d '\n')
+    kubectl apply -f https://cloud.weave.works/k8s/net?k8s-version=$kubever
```

Comment:
one of the reasons for putting this into artifact is to avoid issues related to:
- incompatible version of weave downloaded and installed
- weave site being down during vagrant up?

Reply:
@kmova , this is the suggested way in the kubeadm docs, do we still want to add this YAML as an artist?

Reply:
I think it helps to store as artifact as it reduces the number of unknowns to the users launching our vagrant VMs.

Reply:
Noticing an issue with the /etc/resolv.conf for centos boxes. The fix for prepare_network.sh does not seem to be retained, working on a fix for this.

Will retain the weave yaml as an artifact as discussed above.

Reply:
@kmova implemented the review comments. Please review the new changes. Have implemented flannel as a cni network for now on the Ubuntu Kubernetes 1.9 cluster.